### PR TITLE
added respective .go files for  setup.sh and init-shard.sh 

### PR DESCRIPTION
**Pull request by RISHIKESHk07**
[PMM-XXXX](https://jira.percona.com/browse/PMM-XXXX) (optional, if ticket reported)
 
- [ #721 ] Links to other linked pull requests (optional).

---

- [x] Tests passed.
- [ ] Fix conflicts with target branch.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

Once all checks pass and the code is ready for review, please add `pmm-review-exporters` team as the reviewer. That would assign people from the review team automatically. Report any issues on our [Forum](https://forums.percona.com).

### CODE REVIEW

File: docker/scripts/init-shard.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		time.Sleep(1 * time.Second)
+	}
+	fmt.Println("Started..")
+
+	fmt.Printf("init-shard.sh time now: %s\n", time.Now().Format("15:04:05"))
+
+	rs1 := os.Getenv("RS1")
+	rs2 := os.Getenv("RS2")
+
+	mongodb11Resolved, _ := resolveHost(mongo11)
```
Comment (JiriCtvrtka) [inline docker/scripts/init-shard.go:54]: Since we returning error from resolveHost it would be nice to print it.

File: docker/scripts/setup.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	configServers := []string{mongo1, mongo2, mongo3}
+	replicaSet := []string{mongo1, mongo2, mongo3}
+
+	err = waitForStartup(client, ctx)
+	if err != nil {
+		fmt.Println("Error waiting for startup:", err)
+		os.Exit(1)
+	}
+
+	switch os.Args[1] {
```
Comment (JiriCtvrtka) [inline docker/scripts/setup.go:87]: Maybe check if os args with index 1 is available.

**Review by JiriCtvrtka — APPROVED** [review]
LGTM, just two small comments.

**Review by BupycHuk — CHANGES_REQUESTED** [review]
Please address comments from linter and fix tests

**Comment by RISHIKESHk07** [comment]
@BupycHuk any additional changes ?

**Pull request closed without merging**